.github/workflows/haskell.yml

```diff
@@ -142,8 +142,9 @@ jobs:
 
     - name: Install fourmolu
       run: |
+        FOURMOLU_VERSION="0.14.0.0"
```

Comment:
I'm surprised this didn't require an `export` :slightly_smiling_face:

Reply:
Exporting is needed only if you want the environment variable be available in the subprocess. Here, however, we just do interpolation directly in the same bash session. See the difference:

```shell
> FOO="foo"
> echo "$FOO"
foo
> bash -c 'echo "$FOO"'

> export BAR="bar"
> echo "$BAR"
bar
> bash -c 'echo "$BAR"'
bar
```